build.ps1

```diff
@@ -119,6 +119,20 @@ Invoke-BuildStep 'Building NuGet.Clients projects - VS15 dependencies' {
 ## Building the VS14 Tooling solution
 Invoke-BuildStep 'Building NuGet.Clients projects - VS14 dependencies' {
         param($Configuration, $ReleaseLabel, $BuildNumber, $SkipRestore, $Fast)
+
+        if (-Not $SkipVS15)
+        {
+            Trace-Log "Cleaning VS14 build after VS15 build"
+            $solutionPath = Join-Path $NuGetClientRoot NuGet.Clients.sln
+            $opts = , $solutionPath
+            $opts += "/p:Configuration=$Configuration", "/t:Clean"
```

Comment:
not sure about powershell syntax, but does += accept a comma separated list of strings ?